pkg/oc/cli/image/info/info.go

```diff
@@ -72,6 +74,9 @@ func (o *InfoOptions) Complete(cmd *cobra.Command, args []string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("info expects at least one argument, an image pull spec")
 	}
+	if err := o.FilterOptions.Validate(); err != nil {
```

Comment:
`Complete` should fill in the necessary bits, not perform validation. Even if you're validating internal structure you should create a separate `Validate` method for `InfoOptions` and call `FilterOptions`' `Validate` there.

Reply:
I am not going to refactor this now.  We regressed and this gets us back to working for the use case at hand.  I'm happy to come back later.

Reply:
Later = post kubecon